src/humancellatlas/data/metadata/helpers/dss.py

```diff
@@ -27,7 +32,18 @@ def download_bundle_metadata(client: DSSClient,
 
     :param version: The version of the bundle. if None, the most recent version of the bundle will be downloaded.
 
-    :param num_workers: the size of the thread pool to use for downloading metadata files in parallel. If None, the
+    :param directurls: A boolean controls whether to include direct-access URLs in the response. This is mutually
+                       exclusive with the presignedurls parameter. Note: including `directurls` and `presignedurls` in
+                       the function call will make the DSS Checkout Service to copy the files in the bundle to the
```

Comment:
"… will cause the DSS to copy metadata and data files in the bundle to another bucket first. That could be time-consuming and/or inefficient for …"